minard/views.py

```diff
@@ -1222,6 +1223,14 @@ def noise_run_detail(run_number):
     else:
         return render_template('noise_run_detail.html', run=0, run_number=run_number)
 
+@app.route('/cssProc')
```

Comment:
Although it isn't really followed by most things in minard, I'd like to have urls be all lowercase and with dashes in the title, so could you update this to be just `/css`?

Reply:
When you say dashes for you mean "-" or "_" just so we are one the same page.

Reply:
If you go with just `/css` you don't need anything. But if you want to keep the proc I would say something like `/css-proc` just because it's much easier to type than camel case or underscores.

Reply:
Yeh sure, I have made it "css-proc" but for some reason the diff hasn't updated. The reason for the "proc" bit is that other things are coming soon.